bugbot/rules/webcompat_sightline.py

```diff
@@ -0,0 +1,78 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Any, Optional
+
+from bugbot import gcp
+from bugbot.bzcleaner import BzCleaner
+
+
+class WebcompatSightline(BzCleaner):
+    whiteboard_entry = "[webcompat:sightline]"
+
+    def __init__(self):
+        super().__init__()
+        self.autofix_changes = {}
```

Comment:
The `autofix_changes` attribute is already initialized in the parent class, we could drop it here:
https://github.com/mozilla/bugbot/blob/a8b33bedcf4c7cf0e0deb5448b2748912ffe9ea7/bugbot/bzcleaner.py#L58